src/client/main.tsx

```diff
@@ -5,17 +5,27 @@ import { GameState } from '../core/game';
 import { ClientController } from './client-controller';
 import { Splash } from './splash';
 
+const GAME_START_TIME = 5;
 enum Stages { SPLASH, LOADING, RUNNING };
-export interface ClientState { stage: Stages; }
+export interface ClientState {
+  stage: Stages;
+  numPlayersInLobby: number;
+  countdownTime: number;
+}
 export class Main extends React.Component<{}, ClientState> {
   gameState: GameState;
   controller: ClientController;
   socket: SocketIOClient.Socket;
   activePlayer: string;
+  countdownTimer: number;
 
   constructor() {
     super();
-    this.state = { stage: Stages.SPLASH };
+    this.state = {
```

Comment:
Lets move this lobby stuff into its own `tsx` file, same reasoning that I gave for the 390 thing.

It could take the `numPlayersInLobby` and max countdown time as props, and then it would do its counting down internally with its own state.